packages/babel-plugin-react-server/.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: node_js
+node_js:
+  - "0.12"
+  - "iojs"
+  - v4
+  - v5
```

Comment:
Does travis do anything with this?  I thought it just looked in the repo root...